gradle/gradle-mvn-push.gradle

```diff
@@ -3,6 +3,7 @@
  */
 
 apply plugin: 'maven'
+apply plugin: 'maven-publish'
```

Comment:
publishing to Maven Local doesn't quite work currently - to be done in a future PR, but this going to be required for it